Sources/LeafKit/LeafParser.swift

```diff
@@ -38,6 +38,11 @@ extension TagDeclaration {
                     var buffer = ByteBufferAllocator().buffer(capacity: 0)
                     buffer.writeString(st)
                     return .raw(buffer)
+                case .keyword(let kw) :
+                    guard kw.isBooleanValued else { fallthrough }
```

Comment:
This is awful code flow, visually. I'd actually prefer you repeated the `throw` here instead, though I also wish you weren't throwing strings.

Reply:
Haven't touched any of the error handling in Parser yet, will 100% replace all the string throws soon as I add a direct case for parsing issues to LeafError. Can you live with it as-is?

Once I can revise Syntax to make tags report acceptable parameter signatures, the switch can be totally dumped anyway as it'll be a simple case of matching parameter types against tag signatures.

Reply:
I can live for now 🙂